test/integration/auth.js

```diff
@@ -0,0 +1,34 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('lib', 'utils')
+
+const it = global.it // for lint
+const describe = global.describe // for lint
```

Comment:
This can be avoided later on, once we get to merging the current feature branches. A known workaround and patch for this exists in https://github.com/TransforMap/data.transformap.co/commit/12ae79a4cfde0d02afc5e1d99d7a0f175638dbfd